app/serializers/group_serializer.rb

```diff
@@ -13,6 +13,7 @@ class GroupSerializer < ActiveModel::Serializer
              :members_can_raise_proposals,
              :members_can_vote,
              :memberships_count,
+             :members_count,
```

Comment:
Okay, but I'd prefer to then throw it away and reimplement it, rather than
putting two different implementations in there. I think we use
memberships_count somewhere, and I'm about to use it in the invite people
modal.

On Thu, Jun 11, 2015 at 6:00 PM, Robert Guthrie <notifications@github.com>
wrote:

> In app/serializers/group_serializer.rb
> <https://github.com/loomio/loomio/pull/2226#discussion_r32191874>:
>
> > @@ -13,6 +13,7 @@ class GroupSerializer < ActiveModel::Serializer
> >               :members_can_raise_proposals,
> >               :members_can_vote,
> >               :memberships_count,
> > +             :members_count,
>
> memberships_count is rubbish and should not be trusted. It's just a
> counter_cache: true which does not take into account suspended, archived.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/2226/files#r32191874>.
>